securedrop_client/logic.py

```diff
@@ -761,7 +761,7 @@ def on_delete_source_success(self, source_uuid: str) -> None:
         """
         Rely on sync to delete the source locally so we know for sure it was deleted
         """
-        self.source_deleted.emit(source_uuid)
+        pass
 
     def on_delete_source_failure(self, e: Exception) -> None:
         if not isinstance(e, (RequestTimeoutError, ServerConnectionError)):
```

Comment:
I just realized that here in the source deletion callback we should really be setting the source widget back to its non-pending state (since the job won't get added back to the queue, and so is not pending), this can be another ticket since this is an issue on `master`. Since it's a rare/unexpected case I don't consider it a release blocker, but still, worth resolving.

Reply:
ah, yes, if the deletion fails due to an error other than a timeout then we should set it back to non-pending. good find!